packages/strapi-plugin-graphql/services/__tests__/data-loaders.test.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const dataLoaders = require('../data-loaders');
+
+describe('dataloader', () => {
+  describe('serializeKey', () => {
+    test('Serializes objects to json', () => {
+      expect(dataLoaders.serializeKey(1928)).toBe(1928);
+      expect(dataLoaders.serializeKey('test')).toBe('test');
+      expect(dataLoaders.serializeKey([1, 2, 3])).toBe('[1,2,3]');
+      expect(dataLoaders.serializeKey({ foo: 'bar' })).toBe('{"foo":"bar"}');
+      expect(dataLoaders.serializeKey({ foo: 'bar', nested: { bar: 'foo' } })).toBe(
+        '{"foo":"bar","nested":{"bar":"foo"}}'
+      );
+    });
+  });
+
+  describe('makeQuery', () => {
+    test('makeQuery single calls findOne', async () => {
+      const uid = 'uid';
+      const find = jest.fn(() => [{ id: 1 }]);
+      const findOne = jest.fn(() => ({ id: 1 }));
+      const filters = { _limit: 5 };
+
+      global.strapi = {
+        query() {
+          return { find, findOne };
+        },
+      };
+
+      await dataLoaders.makeQuery(uid, { single: true, filters });
+
+      expect(findOne).toHaveBeenCalledWith(filters, []);
```

Comment:
You mocked the find function but don't use it no ? You may check that it is not called